config/filters.json

```diff
@@ -217,6 +217,7 @@
         "^золото(й)?$",
         "^зоо(магазин|товары)$",
         "^канцтовары$",
+        "^касса$"
```

Comment:
There needs to be a comma after the regex. Otherwise the build does not work.

Reply:
my bad, comma is there now